src/backend/common/helpers/insights_helper.py

```diff
@@ -324,6 +327,46 @@ def _calculate_leaderboard_most_matches_played_by_team(
             )
         ]
 
+    @classmethod
+    def _calculate_highest_clean_and_combined_scores(
+        cls, week_event_matches: List[WeekEventMatches], year: Year
+    ) -> List[Insight]:
+        """Core logic is pretty much the same for both insights, so do both in 1 method."""
+        match_winning_scores = defaultdict(int)
+        match_combined_scores = defaultdict(int)
+
+        for _, week_events in week_event_matches:
+            for _, matches in week_events:
+                for match in matches:
+                    if match.has_been_played:
+                        redScore = int(match.alliances[AllianceColor.RED]["score"])
+                        blueScore = int(match.alliances[AllianceColor.BLUE]["score"])
+
+                        if year >= 2016:
```

Comment:
I took this logic from another function in this file, but it used `year >= 2017` -- I checked the 2016 score breakdowns and it works there too. Was there a reason it was 2017? @fangeugene

Reply:
no clue 🤷